docs/src/getting_started.md

```diff
@@ -0,0 +1,137 @@
+# Getting Started with Chmy.jl
+
+[Chmy.jl](https://github.com/PTsolvers/Chmy.jl) is powered by [KernelAbstractions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl) and it is a backend-agnostic toolkit for finite difference computations on multi-dimensional computational staggered grids. In this introductory tutorial, we will show the essence of Chmy.jl in which we resolved a simple 2D diffusion problem. The full code of the tutorial material is available under [diffusion_2d.jl](https://github.com/PTsolvers/Chmy.jl/blob/main/examples/diffusion_2d.jl).
+
+## Basic Diffusion
```

Comment:
Maybe it would actually make more sense to keep `C` but use diffusion of concentration (Fick's law). As such, we do not need to modify the variable names everywhere